proto/configpb.proto

```diff
@@ -12,7 +13,9 @@ option (rustproto.lite_runtime_all) = true;
 service Config {
     rpc Create(CreateRequest) returns (CreateResponse) {}
     rpc Get(GetRequest) returns (GetResponse) {}
```

Comment:
Please add gateway for other methods too.

Reply:
This PR is only for compatibility.